extruct/_extruct.py

```diff
@@ -134,7 +135,7 @@ def extract(htmlstring,
         for syntax, uniform, raw, schema_context in uniform_processors:
             try:
                 if syntax == 'opengraph':
-                    output[syntax] = uniform(raw)
+                    output[syntax] = uniform(raw, with_og_array)
                 else:
                     output[syntax] = uniform(raw, schema_context)
```

Comment:
Right makes sense, so I'll have to change both `_uopengraph` and `_umicrodata_microformat` right ?

Reply:
> I'll have to change both _uopengraph and _umicrodata_microformat right ?

I think no changes there are required, because we pass ``schema_context`` only to ``_umicrodata_microformat``, and ``with_og_array`` only to ``_uopengraph`` - I just suggested to make it more clear that these are different arguments, as opposed to different values of the same arguments, sorry that I was not clear.

Reply:
Oh my bad I got it confused with `**kwargs`